modules/machineset-yaml-vsphere.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+// * machine_management/creating_machinesets/creating-machineset-vsphere.adoc
+// * post_installation_configuration/cluster-tasks.adoc
+
+[id="machineset-yaml-vsphere_{context}"]
+=  Sample YAML for a MachineSet Custom Resource on vSphere
+
+This sample YAML defines a MachineSet that runs on VMware vSphere and creates nodes that are labeled with `node-role.kubernetes.io/<role>: ""`.
+
+In this sample, `<infrastructureID>` is the infrastructure ID label that is based on the cluster ID that you set when you provisioned the cluster, and `<role>` is the node label to add.
+
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  creationTimestamp: null
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+  name: <infrastructureID>-<role> <2>
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+    machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+    machine.openshift.io/cluster-api-machineset: <infrastructureID>-<role> <2>
+template:
+    metadata:
+    creationTimestamp: null
+    labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+        machine.openshift.io/cluster-api-machine-role: <role> <3>
+        machine.openshift.io/cluster-api-machine-type: <role> <3>
+        machine.openshift.io/cluster-api-machineset: <infrastructureID>-<role> <2>
+    spec:
+    metadata:
+        creationTimestamp: null
+        labels:
+            node-role.kubernetes.io/<role>: "" <3>
+    providerSpec:
+        value:
+        apiVersion: vsphereprovider.openshift.io/v1beta1
+        credentialsSecret:
+            name: vsphere-cloud-credentials
+        diskGiB: 120
+        kind: VSphereMachineProviderSpec
+        memoryMiB: 8192
+        metadata:
+            creationTimestamp: null
+        network:
+            devices:
+            - networkName: "<vm-network-name>" <4>
+        numCPUs: 4
+        numCoresPerSocket: 1
+        snapshot: ""
+        template: <vm-template-name> <5>
+          userDataSecret:
+            name: worker-user-data
+          workspace:
+            datacenter: <vcenter-dataCenter-name> <6>
+            datastore: <vcenter-datastore-name> <7>
+            folder: <vcenter-vm-folder-path> <8>
+            resourcepool: <vsphere-resource-pool> <9>
+            server: <vcenter-server-ip> <10>
+----
+<1> Specify the infrastructure ID that is based on the cluster ID that
+you set when you provisioned the cluster. If you have the OpenShift CLI and `jq`
```

Comment:
The following command doesn't appear to use `jq`, it uses jsonpath (which is preferred to jq), so I don't think you need this note about jq here.